src/main/java/com/liberty/dao/SearchBookDAO.java

```diff
@@ -0,0 +1,54 @@
+package com.liberty.dao;
+
+import com.liberty.model.FullBookEntity;
+
+/**
+ * Created by user on 12.05.2017.
+ */
+public class SearchBookDAO {
```

Comment:
Reneame to Search Book entity. DAO it is repository. And use lombok @Data annotation instead of getters and setters, equals and hashcode